builtin/providers/aws/config.go

```diff
@@ -213,6 +215,9 @@ func (c *Config) Client() (interface{}, error) {
 
 		log.Println("[INFO] Initializing CodeDeploy Connection")
 		client.codedeployconn = codedeploy.New(sess)
+
+		log.Println("[INFO] Initializing CodeCommit SDK connection")
+		client.codecommitconn = codecommit.New(awsConfig)
```

Comment:
Hm - did codecommit not get the new session.Session refactoring that the rest of the connections did this week? I'm surprised this builds!

I'd expect the line here to need `usEast1Sess` as the argument.